x-pack/plugins/ml/server/lib/check_license/check_license.ts

```diff
@@ -49,10 +49,11 @@ export function checkLicense(xpackLicenseInfo: XPackInfo): Response {
 
   const VALID_FULL_LICENSE_MODES = ['platinum', 'enterprise', 'trial'];
```

Comment:
See other comment regarding moving this into a common constants file. Could be done in the follow-up to fix the licensing checks.

Reply:
609be5c4c218d1feed4b249158c438ce7c30f99d